config/prow/cluster/boskos_test.go

```diff
@@ -1,34 +0,0 @@
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package cluster
-
-import (
-	"testing"
-
-	"sigs.k8s.io/boskos/common"
-)
-
-func TestConfig(t *testing.T) {
```

Comment:
this should probably be replaced with a presubmit that verifies the config: https://github.com/openshift/release/blob/94da441c473adf1dc6e4c76231b64b6f4a4f9d49/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml#L41-L65

Reply:
Ah I just realized what this test was actually doing - you are right, looks like we shouldn't just remove it.

BTW, an alternative is to remove it now, then once boskos updates we can add it back. It just depends if we want to stop importing boskos forever, or just to avoid the bad client-go import temporarily. Probably long term its good to not have a circular dependency.